core/src/constants/COLLATERALS.ts

```diff
@@ -90,6 +90,10 @@ const COLLATERALS: Record<string, CollateralConfig> = {
         symbol: 'ETH',
         decimals: 18,
         exchanges: {
+            'Uniswap V3 Autorouter': {
+                callee: 'UniswapV3Callee',
+                automaticRouter: true,
+            },
```

Comment:
Can you please add the same market to all other collaterals with `UniswapV3Callee`?

Reply:
This wasn't addressed so far

Reply:
i am extremely confused because i pasted the config and it's not there now. I need a christmas vacation apparently